misk/src/test/kotlin/misk/config/ConfigTest.kt

```diff
@@ -18,9 +15,10 @@ import javax.inject.Named
 
 @MiskTest
 class ConfigTest {
+    val config = MiskConfig.load<TestConfig>(TestConfig::class.java, "test_app", TESTING)
```

Comment:
it can't infer `TestConfig`?

Reply:
It should. But also, see above comments on using an inlined reified version for this.